github/Organization.py

```diff
@@ -673,6 +675,35 @@ def add_to_public_members(self, public_member: NamedUser) -> None:
             "PUT", f"{self.url}/public_members/{public_member._identity}"
         )
 
+    def get_credential_authorizations(self) -> PaginatedList[AuthorizationOrganization]:
+        """
+        :calls: `GET /orgs/{org}/credential-authorizations <https://docs.github.com/en/enterprise-cloud@latest/rest/orgs/orgs#list-saml-sso-authorizations-for-an-organization>`_
+        :rtype: :class:`PaginatedList` of :class:`github.AuthorizationOrganization.AuthorizationOrganization`
+
+        List SAML SSO authorizations for an organization.
+        """
+        return PaginatedList(
+            github.AuthorizationOrganization.AuthorizationOrganization,
+            self._requester,
+            f"{self.url}/credential-authorizations",
+            None,
+            headers={"Accept": Consts.mediaType},
+        )
+
```

Comment:
We should also have a

    def get_credential_authorizations(self, credential_id: int) -> AuthorizationOrganization: